scripts/zkai_attention_kv_d16_native_width_gate.py

```
@@ -0,0 +1,626 @@
+    if summary["required_backend_version"] != required_backend_version:
+        raise AttentionKvD16NativeWidthGateError("backend version drift")
+    if summary["key_width"] != key_width or summary["value_width"] != value_width:
+        raise AttentionKvD16NativeWidthGateError("width drift")
+    if summary["sequence_length"] != 8:
+        raise AttentionKvD16NativeWidthGateError("sequence length drift")
+    if summary["score_row_count"] != 52 or summary["trace_row_count"] != 64:
+        raise AttentionKvD16NativeWidthGateError("row count drift")
+    if summary["final_kv_items"] != 10:
+        raise AttentionKvD16NativeWidthGateError("final KV item count drift")
+    if summary["selected_positions"] != list(selected_positions):
+        raise AttentionKvD16NativeWidthGateError("selected positions drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)